Resources/Locale/en-US/chat/emotes.ftl

```diff
@@ -4,7 +4,7 @@ chat-emote-name-laugh = Laugh
 chat-emote-name-honk = Honk
 chat-emote-name-sigh = Sigh
 chat-emote-name-whistle = Whistle
-chat-emote-name-crying = Crying
+chat-emote-name-crying = Cry
```

Comment:
I noticed this while testing -- nearly all of the other emotes are in the infinitive tense rather than the gerund like `Crying` here.
(okay if you wanna fight me, it's probably more accurately described as the imperative mood without a person-conjugation, but the point remains that it was wrong)